### ISSUE TITLE
Not working on Big Sur and later

### ISSUE THREAD

**Issue by a7madgamal** [sindresorhus/do-not-disturb-cli#2]
Doesn't seem to have any effect although it reports everything is fine and status command reports the expected result (according to my usage)

**Comment by alexreg** [comment]
Yep, same here. The program seems to have its own version of the Do Not Disturb status that is quite independent from the macOS system status.

**Comment by abaj8494** [comment]
has any one come up with an alternative solution? perhaps using applescript?

**Comment by strafe** [comment]
@br1ttle check out https://github.com/vitorgalvao/tiny-scripts/blob/master/calm-notifications.
```
brew install vitorgalvao/tiny-scripts/calm-notifications
```

**Comment by karlhorky** [comment]
Probably related to the Big Sur / Monterey breakage in the underlying `do-not-disturb` module: https://github.com/sindresorhus/do-not-disturb/issues/9

There are workarounds over there, if you need them.

**Comment by gqbre** [comment]
Not working on Monterey too

**Comment by ryan6416** [comment]
Is there a working alternative for Monterey?

**Comment by gqbre** [comment]
@ryan-efendy https://github.com/sindresorhus/do-not-disturb/issues/9#issuecomment-985931317

**Comment by sprzedwojski** [comment]
Hey folks, this solution worked quite nicely for me: https://github.com/sindresorhus/do-not-disturb/issues/9#issuecomment-991956241

It's this repo: https://github.com/arodik/macos-focus-mode

It uses a MacOS shortcut but has a pretty nice CLI.

I will be replacing `do-not-disturb-cli` with it in my Pomodoro project: https://github.com/sprzedwojski/pymodoro

**Comment by sindresorhus** [comment]
You are right, and this is not fixable. macOS 11 replaced Do Not Disturb with Focus, and Apple ships no public API to set it. I checked every route: the private `DoNotDisturb` framework needs an entitlement that third-party apps cannot get, the private database needs Full Disk Access, and Shortcuts can set Focus but has no action to read the current state, so `status` and `toggle` cannot exist at all.

Deprecating the package and archiving the repo. Use a shortcut with the built-in "Set Focus" action, then `shortcuts run "<shortcut name>"`.
